File: API/Models/Report_Sort_SumResult.cs

using System.ComponentModel.DataAnnotations.Schema;

namespace API.Models
{
    public partial class Report_Sort_SumResult
    {
        public DateTime? crday { get; set; }
        public DateTime mdat { get; set; }
        public string brandname { get; set; }
        public string cusid { get; set; }
        public string cusna { get; set; }
        public string manno { get; set; }
        public string purno { get; set; }
        public string rmodel { get; set; }
        public string tolcls { get; set; }
        public string ritnbr { get; set; }
        public string bitnbr { get; set; }
        public string article { get; set; }
        public string kind { get; set; }
        public DateTime? eta { get; set; }
        public string size { get; set; }
        [Column(TypeName = "decimal(7, 1)")]
        public decimal? qty { get; set; }
        [Column(TypeName = "decimal(7, 1)")]
        public decimal? wkshqty { get; set; }
        [Column(TypeName = "decimal(7, 1)")]
        public decimal? pqty { get; set; }
        [Column(TypeName = "decimal(7, 1)")]
        public decimal sort_qty { get; set; }
        [Column(TypeName = "decimal(7, 1)")]
        public decimal? diff_qty { get; set; }
        [Column(TypeName = "decimal(7, 1)")]
        public decimal? cqty { get; set; }
    }
}

File: SPA/src/app/_core/models/Report/reportSortSumResult.ts
export interface ReportSortSumResult {
  crday: string | null;
  mdat: string;
  brandname: string;
  cusid: string;
  cusna: string;
  manno: string;
  purno: string;
  rmodel: string;
  tolcls: string;
  ritnbr: string;
  bitnbr: string;
  article: string;
  kind: string;
  eta: string | null;
  size: string;
  qty: number | null;
  wkshqty: number | null;
  pqty: number | null;
  sort_qty: number;
  diff_qty: number | null;
  cqty: number | null;
  check: boolean
}

export interface SearchSortSumReportParams {
  crday: string;
  dateStart: string;
  dateEnd: string;
  brandname: string;
  cusna: string;
  manno: string;
  tolcls: string;
  rmodel: string;
  purno: string;
  bitnbr: string;
  kind: string;
  etaFrom: string;
  etaTo: string;
  size: string;
}
